WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsTrackerWrapper.kt

```diff
@@ -60,5 +60,13 @@ class AnalyticsTrackerWrapper
         AnalyticsUtils.trackWithSiteDetails(stat, siteModel)
     }
 
+    fun trackWithSiteDetails(
+        stat: Stat,
+        siteModel: SiteModel?,
+        properties: Map<String, Any>
```

Comment:
This is more of a suggestion than a change request: What if we added the `properties` parameter with a default value `null` in the function above?

Reply:
Oh, your comment made me realize that this (Stat, SiteModel, properties) version of the function is not used anywhere and it's probably code from an intermediate state of my effort here. I shouldn't have committed it. WDYT about removing it altogether?

While at it, it looks like at this stage, [this version of `trackWithEvents`](https://github.com/wordpress-mobile/WordPress-Android/blob/94ceccabb0fa240a87518301d8a002f30f44185f/WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsTrackerWrapper.kt#L56-L61) has the exact same signature and implementation as [this `track` version](https://github.com/wordpress-mobile/WordPress-Android/blob/94ceccabb0fa240a87518301d8a002f30f44185f/WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsTrackerWrapper.kt#L41-L43) anyway. WDYT about letting `trackWithEvent` go in favor or a `track` with an optional `properties` parameter? It would look like this: https://github.com/wordpress-mobile/WordPress-Android/compare/gb/add-blog-id-to-editor-session-events...gb/add-blog-id-to-editor-session-events-consolidated-functions

Reply:
> WDYT about letting trackWithEvent go in favor or a track with an optional properties parameter?

Thats exactly what I was thinking of but I hadn't noticed the method with the exact same signature 👍

Reply:
Ah, perfect, pushed 8613b41 with the changes.

Reply:
Thank you for the change Stefanos 👍